cumulusci.yml

```diff
@@ -43,3 +87,41 @@ flows:
         task: deploy_dev_config
       4:
         task: load_dataset
+
+  dependencies:
+    steps:
+      3:
+        task: ensure_record_type
+
+plans:
+  existing_org:
+    slug: install
+    title: Install Ombudsman Cloud Care
+    is_listed: True
+    tier: primary
+    preflight_message: "This will install Ombudsman Cloud Care into your org."
+    post_install_message: "Thanks for installing Ombudsman Cloud Care. Please visit the [Ombudsman Cloud Care](https://powerofus.force.com/s/group/0F91E0000000hq1SAA) community group on the Power of Us Hub for any questions about Ombudsmand Cloud Care."
+    error_message: "To get help with this error, go to [Ombudsman Cloud Care](https://powerofus.force.com/s/group/0F91E0000000hq1SAA) community group on the Power of Us Hub and post a question include any details about the install."
+
+    steps:
+      1:
+        task: update_dependencies
+      2:
+        task: deploy
```

Comment:
You can use `deploy_pre` here and CCI will take care of the options. `unpackaged/pre` should contain one or more bundle subdirectories, and CCI will deploy each of them.

Reply:
Groovy. I'll update.